File: SORTS/BubbleSort.java
import java.util.Scanner;
public class BubbleSort {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int a [] = new int[20];

        System.out.print("How many element do you want to enter:");
        int n = sc.nextInt();
        for(int i=0; i<n; i++){
            a[i] = sc.nextInt();
        }

        System.out.println("Your array elements before sorting ");
        for(int i=0; i<n; i++){
            System.out.println(a[i]);
        }

        for(int i=0; i<n-1; i++){
            for(int j=0; j<n-1; j++){
                if(a[j]>a[j+1]){
                 int temp = a[j];
                 a[j] = a[j+1];
                 a[j+1] = temp;
                }
            }
        }

        System.out.println("Your array elements after sorting ");
        for(int i=0; i<n; i++){
            System.out.println(a[i]);
        }
    }
}

File: SORTS/Bubblesort.c
#include<stdio.h>
int main()
{
int a[50],n,i,j,k,l,temp;
printf("This is the program of bubble sort");
printf("\nHow many element do you want to enter:");
scanf("%d",&n);
printf("\nEnter the elements:");
for(i=0; i<n; i++)
{
    scanf("%d",&a[i]);
}
printf("\nThe array element are:");
for(j=0; j<n; j++)
{
    printf(":%d",a[j]);
}

for(j=0; j<n; j++)
{
    for(k=0; k<n-1; k++)
    {
        if(a[k]>a[k+1])
        {
          temp=a[k];
          a[k]=a[k+1];
          a[k+1]=temp;
        }
    }
}
printf("\nYour array elements are sorted");
for(l=0; l<n; l++)
{
    printf(":%d",a[l]);
}



return 0;


}